lib/board.rb

```diff
@@ -130,10 +122,44 @@ def place(ship, coords)
 
   #OPTIMIZE - Will refactor if possible
   def render(show_ship = false)
-    if show_ship == true
-      return "  1 2 3 4 \nA #{@cells["A1"].render(true)} #{@cells["A2"].render(true)} #{@cells["A3"].render(true)} #{@cells["A4"].render(true)} \nB #{@cells["B1"].render(true)} #{@cells["B2"].render(true)} #{@cells["B3"].render(true)} #{@cells["B4"].render(true)} \nC #{@cells["C1"].render(true)} #{@cells["C2"].render(true)} #{@cells["C3"].render(true)} #{@cells["C4"].render(true)} \nD #{@cells["D1"].render(true)} #{@cells["D2"].render(true)} #{@cells["D3"].render(true)} #{@cells["D4"].render(true)} \n"
-    else
-      return "  1 2 3 4 \nA #{@cells["A1"].render} #{@cells["A2"].render} #{@cells["A3"].render} #{@cells["A4"].render} \nB #{@cells["B1"].render} #{@cells["B2"].render} #{@cells["B3"].render} #{@cells["B4"].render} \nC #{@cells["C1"].render} #{@cells["C2"].render} #{@cells["C3"].render} #{@cells["C4"].render} \nD #{@cells["D1"].render} #{@cells["D2"].render} #{@cells["D3"].render} #{@cells["D4"].render} \n"
+    board_string = " "
+    count = 1
+    num_array = []
+
+    @width.times do
+      board_string << " #{count}"
+      num_array << count
+      count += 1
+    end
+
+    @letters.each do |letter|
+      board_string << " \n#{letter.upcase}"
+      num_array.each do |number|
+        if show_ship == true
+          board_string << " #{@cells["#{letter.upcase + number.to_s}"].render(true)}"
+        else
+          board_string << " #{@cells["#{letter.upcase + number.to_s}"].render}"
+        end
+      end
+    end
+    board_string
+  end
+
+  def get_board_size
+    p "What height would you like the board to be? (26 max)"
+    height = gets.chomp.to_i
+    p "What width would you like the board to be?"
+    @width = gets.chomp.to_i
+    alphabet = ('a'..'z').to_a
+    @letters = alphabet[0..height - 1]
+    letters.each do |letter|
+      column = 1
+      width.times do
+        coord = letter + column.to_s
+        @cells[coord.upcase] = Cell.new(coord.upcase)
+        coord = ''
+        column += 1
+      end
```

Comment:
Each number, letter, and cell render is appended to the board_string for printing purposes